lib/brakeman/util.rb

```diff
@@ -124,6 +124,18 @@ def hash_access hash, key
     nil
   end
 
+  def call_has_param call, key
```

Comment:
Hum. Not sure if I quite understood what you meant. Where do we check if it's calling the function `:to_unsafe_hash` in your example? In my example, I am using the `key` parameter to do that, but I am cannot see something similar in yours. Could you please elaborate a bit so I can proper fix it? :smile: